scripts/bootstrap-build-framework.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+script_path=$(dirname $0)
+working_dir=$(pwd)
+cd "$script_path"
+
+if [ ! -d "./make" ]; then
+  version="master"
+  if [ -f ./.dbf ]; then
+    version=$(cat ./.dbf)
```

Comment:
is this an optional file that can be defined against a project to set the version for future proofing the framework?

Reply:
yup